plugin/pom.xml

```diff
@@ -212,7 +205,7 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-assembly-plugin</artifactId>
                 <configuration>
-                    <finalName>${project.artifactId}-${project.version}-ES-${elasticsearch.version}</finalName>
```

Comment:
Because it's in the version of the plugin (as is standard for all of our ES plugins). See https://github.com/rosette-api/rosette-elasticsearch-plugin#note-on-versioning